bokehjs/test/unit/models/widgets/checkbox_button_group.ts

```diff
@@ -33,7 +33,7 @@ describe("CheckboxButtonGroup", () => {
       expect(g.active).to.be.equal([0, 1, 2])
 
       const button = view.shadow_el.querySelector<HTMLElement>(".bk-btn:nth-child(1)")
-      button!.click()
```

Comment:
This was throwing lint errors for me - 
```
36:7  error  This assertion is unnecessary since it does not change the type of the expression  @typescript-eslint/no-unnecessary-type-assertion
```

Not sure why the check was added. Let me know if I should drop this change.

Reply:
It's possible it was needed in the past but not any longer, combined with different local versions of tools complaining about different things. If the tests now runs and passes without the `!` that is better.

Reply:
Seems that it does not, though:
```
test/unit/models/widgets/checkbox_button_group.ts:36:7 - error TS2531: Object is possibly 'null'.

36       button.click()
         ~~~~~~
``` 
I would say restore it for now

Reply:
`!` is definitively necessary given `querySelector` returns a value of type `T | null`.